pho/src/components/RegisterContainer/Register.tsx

```diff
@@ -63,6 +66,15 @@ export default function Register({ onOpenLogin }: RegisterProps) {
       const hashedPassword = sha256(password + salt);
 
       await registerViaAPI(username, salt, hashedPassword);
+      await toast({
+        title: 'Account created successfully.',
+        description:
+          "We've created your account for you. Please sign in to continue",
+        status: 'success',
+        duration: 9000,
+        isClosable: true,
+      });
```

Comment:
`9000` milliseconds is quite long for a pop-up to be shown in the main screen. Can you adjust the time to be around 3000ms-4000ms? @thienanhvu

Reply:
Resolved! Thank you